pkg/reconciler/reconciler.go

```diff
@@ -746,6 +746,12 @@ func (r *Reconciler) doInstall(actionClient helmclient.ActionInterface, u *updat
 	r.reportOverrideEvents(obj)
 
 	log.Info("Release installed", "name", rel.Name, "version", rel.Version)
+
+	// If log verbosity is higher, output Helm Release Manifest
+	if log.V(1).Enabled() {
```

Comment:
Do we log anything else at the V(1) level? These manifests can be really long, so I wonder if we should bump this number up a bit higher, maybe like 4 or 5?

Reply:
I didn't check that. I will take a deeper dive into that and see.

Reply:
bumped it to V(4) in commit https://github.com/operator-framework/helm-operator-plugins/pull/162/commits/8e6aae322d41c8589b56336e506b31f26e05b4c0